src/_press/cal-itp-benefits-launch.md

```diff
@@ -87,7 +87,7 @@ pleasure.”
 
 For more information about [Cal-ITP Benefits](https://benefits.calitp.org/), contact [hello@calitp.org](mailto:hello@calitp.org).
 
-## About this demonstration project’s stakeholders
+## <u>About this demonstration project’s stakeholders</u>
```

Comment:
Can we do this with CSS instead? 

Something like 

```css
.press-release h2:last-of-type {
  text-decoration: underline;
}
```

I'd rather we didn't have to remember to add this to each Press Release, and they should all be consistent.

Reply:
Done: https://github.com/cal-itp/calitp.org/pull/150/commits/49b6ab4e013c7b92292d357bbf5dc58b97237309